app.rb

```diff
@@ -0,0 +1,22 @@
+require 'sinatra/base'
+require_relative 'lib/peep'
+
+# controller
+class Chitter < Sinatra::Base
+  get '/' do
+    @peepdeck = Peep.all
+    puts @peepdeck
```

Comment:
I think at the end this line can be taken out of the project

Reply:
agreed, was there for visibility but forgot to take it out